examples/keras_io/vision/image_classification_with_vision_transformer.py

```diff
@@ -105,32 +112,52 @@ def __init__(self, patch_size):
         self.patch_size = patch_size
 
     def call(self, images):
-        batch_size = tf.shape(images)[0]
-        patches = tf.image.extract_patches(
```

Comment:
We have an open PR to add `extract_patches` to the API, so I suggest we wait for it and use it in the example

Reply:
This is merged now, you should be able to use it.